### ISSUE TITLE
.ci/deps.sh: wget "Unable to establish SSL connection." when downloading astyle from sourceforge

### ISSUE THREAD

**Issue by refeed** [coala/coala-bears#2552]
https://travis-ci.org/coala/coala-bears/jobs/396784107#L2256

```
+wget 'https://downloads.sourceforge.net/project/astyle/astyle/astyle%203.0.1/astyle_3.0.1_linux.tar.gz?r=&ts=1499017588&use_mirror=excellmedia' -O /home/travis/astyle.tar.gz
--2018-06-26 09:24:46--  https://downloads.sourceforge.net/project/astyle/astyle/astyle%203.0.1/astyle_3.0.1_linux.tar.gz?r=&ts=1499017588&use_mirror=excellmedia
Resolving downloads.sourceforge.net (downloads.sourceforge.net)... 216.105.38.13
Connecting to downloads.sourceforge.net (downloads.sourceforge.net)|216.105.38.13|:443... connected.
Unable to establish SSL connection.
```

Somehow, wget can't connect to https://downloads.sourceforge.net/project/astyle/astyle/astyle%203.0.1/astyle_3.0.1_linux.tar.gz?r=&ts=1499017588&use_mirror=excellmedia due to SSL problem..
I think the link needs to be updated to: https://downloads.sourceforge.net/project/astyle/astyle/astyle%203.0.1/astyle_3.0.1_linux.tar.gz?r=&ts=1530014759&use_mirror=excellmedia

**Comment by jayvdb** [comment]
see also https://github.com/coala/coala-bears/issues/2302

**Comment by jayvdb** [comment]
When I run this command on my laptop, it works correctly.

```console
$ wget 'https://downloads.sourceforge.net/project/astyle/astyle/astyle%203.0.1/astyle_3.0.1_linux.tar.gz?r=&ts=1499017588&use_mirror=excellmedia' -O astyle.tar.gz
--2018-06-27 19:14:22--  https://downloads.sourceforge.net/project/astyle/astyle/astyle%203.0.1/astyle_3.0.1_linux.tar.gz?r=&ts=1499017588&use_mirror=excellmedia
Resolving downloads.sourceforge.net (downloads.sourceforge.net)... 216.105.38.13
Connecting to downloads.sourceforge.net (downloads.sourceforge.net)|216.105.38.13|:443... connected.
HTTP request sent, awaiting response... 302 Found
Location: https://excellmedia.dl.sourceforge.net/project/astyle/astyle/astyle%203.0.1/astyle_3.0.1_linux.tar.gz [following]
--2018-06-27 19:14:29--  https://excellmedia.dl.sourceforge.net/project/astyle/astyle/astyle%203.0.1/astyle_3.0.1_linux.tar.gz
Resolving excellmedia.dl.sourceforge.net (excellmedia.dl.sourceforge.net)... 202.153.32.19
Connecting to excellmedia.dl.sourceforge.net (excellmedia.dl.sourceforge.net)|202.153.32.19|:443... connected.
HTTP request sent, awaiting response... 200 OK
Length: 172483 (168K) [application/x-gzip]
Saving to: ‘astyle.tar.gz’

astyle.tar.gz                             100%[===================================================================================>] 168.44K   122KB/s    in 1.4s    

2018-06-27 19:14:48 (122 KB/s) - ‘astyle.tar.gz’ saved [172483/172483]

```

Is it working for you on your laptop?
Why is it failing on Travis?
Maybe an older version of wget with incompatible SSL?

**Comment by refeed** [comment]
@jayvdb Hmm interesting, it was not working when I submitted this issue. Yes, it's working on my laptop right now... It's kind of flaky..

>  Maybe an older version of wget with incompatible SSL?

I'm not sure, it was very flaky at the time I submitted this issue. It was like I got `Unable to establish SSL connection` and succeed downloaded it back and forth.

Like you said, creating a OBS or PPA for this package would be a long term and ideal solution for this
